tests/meta/transfer/test_external_data.py

```diff
@@ -130,13 +129,13 @@ def test_add_files_to_bundles(
     # GIVEN a file to be added
 
     # WHEN the files are added.
-    external_data_api.add_files_to_bundles(
+    external_data_api.add_and_include_files_to_bundles(
```

Comment:
Also change name of test to reflect this.

Reply:
done